csrc/device_lower/lower2device.cpp

```diff
@@ -160,7 +160,6 @@ class ConvertAlignedBlockSync : kir::IrVisitor {
                 RegisterType::GeneralPurpose) {
           return;
         }
-        return;
```

Comment:
@zasdfgbnm I believe this should not be here, right?

Reply:
I'll remove this function as I added a copy in ir_utils. Just want to make sure this `return` is not necessary.

Reply:
I think your understanding is correct. This fix do change behavior, and fortunately we don't see perf regression on it :)

Reply:
Thanks for confirmation. I'll do a quick cleanup

Reply:
Are you planning to remove this in this PR, or do it in a separate PR?

Reply:
Done. Running the tests locally just in case